cli/mender-artifact/copy.go

```diff
@@ -22,17 +22,12 @@ import (
 	"path/filepath"
 	"regexp"
 
-	"github.com/mendersoftware/mender-artifact/artifact"
 	"github.com/urfave/cli"
 )
 
 var isimg = regexp.MustCompile(`\.(mender|sdimg|uefiimg|img)`)
 
 func Cat(c *cli.Context) (err error) {
-	comp, err := artifact.NewCompressorFromId(c.GlobalString("compression"))
-	if err != nil {
-		return cli.NewExitError("compressor '"+c.GlobalString("compression")+"' is not supported: "+err.Error(), 1)
-	}
```

Comment:
I think this block (and the other ones you changed) should be replaced with a warning instead then. Something like "`--compression flag ignored, using existing compression type`" or something like that.

Reply:
Hmm, yeh, maybe. This one's a little annoying too though, as the flag always has a default `gzip` value, has it not? So the warning will be printed on every single invocation

Reply:
Yeah, but maybe it can be changed to empty default instead? Which is taken into account when the compressor is created.